code/game/objects/items/weapons/lighters.dm

```diff
@@ -1,7 +1,7 @@
-// Basic lighters
+// MARK: BASIC LIGHTERS
```

Comment:
```suggestion
// MARK: LIGHTERS
```
We should change this to `LIGHTERS`, then you can use a mark to designate the matches. That way we are not overusing mark.